app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
 from app.main import get_human_age
+from typing import Any
```

Comment:
Order your imports and separate different groups of them with one blank line according to PEP-8.